rest/blip_sync.go

```diff
@@ -308,7 +308,8 @@ func (bh *blipHandler) sendBatchOfChanges(changeArray [][]interface{}) {
 		bh.sender.Send(outrq)
 	}
 	if len(changeArray) > 0 {
-		base.LogTo("Sync", "Sent %d changes to client, from seq %v ... %s", len(changeArray), changeArray[0][0], bh.effectiveUsername)
+		sequence := changeArray[0][0].(db.SequenceID)
```

Comment:
This wasn't strictly required, but it made things easier to understand.  At some point I'd like to introduce stronger typing for the changes struct:

> Each change in the array is encoded as a nested array of the form [sequence, docID, revID, deleted], i.e. sequence ID followed by document ID followed by revision ID followed by the deletion state (which can be omitted if it's false.)

So I imagine this will be a lot clearer anyway once that's done.